front-end/src/renderer/components/Transaction/Create/NodeCreate/NodeFormData.vue

```diff
@@ -21,6 +22,14 @@ const gossipIpOrDomain = ref('');
 const serviceIpOrDomain = ref('');
 const gossipPort = ref('');
 const servicePort = ref('');
+const publicKeyHash = ref('');
+const hash = ref('');
+const grpcCertificate = ref('');
+const gossipCaCertificate = ref('');
+const gossipInput = useTemplateRef('gossipInput');
+const grpcInput = ref<HTMLInputElement | null>(null);
```

Comment:
The `useTemplateRef` is a new feature of Vue (3.5 and above). Since you are using it for the `gossipInput`, may you also use it for the `grpcInput`, `gossipFile`, and `grpcFile`?